presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -370,6 +371,10 @@ baseType
     | identifier
     ;
 
+intervalType
```

Comment:
Inline this in the `type` rule above. There's no real benefit in keeping it separated.